crates/storage/nippy-jar/src/lib.rs

```diff
@@ -514,6 +514,8 @@ impl DataReader {
         // Ensure that the size of an offset is at most 8 bytes.
         if offset_size > 8 {
             return Err(NippyJarError::OffsetSizeTooBig { offset_size })
+        } else if offset_size == 0 {
```

Comment:
this passes if the offset is at least 1 byte, but the error says "at least 2 bytes"

Reply:
That's typo. The error should be "at least 1 bytes". I've updated it.